airflow/providers/amazon/aws/operators/athena.py

```diff
@@ -38,8 +38,8 @@ class AWSAthenaOperator(BaseOperator):
     :type database: str
     :param output_location: s3 path to write the query results into. (templated)
     :type output_location: str
-    :param aws_conn_id: aws connection to use
-    :type aws_conn_id: str
+    :param aws_conn_id: Aws connection to use in template_fields so that it can be overriden using jinja template from Dags. 
```

Comment:
Thanks for updating this, it's looking pretty good now. Although my preference would be to follow what the other templated fields do, which is just to append `(templated)` to the original doc string.

Reply:
done.